scripts/start_emulator.sh

```
@@ -15,14 +15,15 @@ fi
 
 echo "✅ Firebase CLI found"
 
-# Navigate to root directory (where firebase.json is located)
+# Navigate to the Firebase repository directory (where firebase.json is located)
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_ROOT="$(dirname "$SCRIPT_DIR")"
-cd "$PROJECT_ROOT/.."
+cd "$PROJECT_ROOT"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)